org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/java/org/eclipse/winery/bpmn2bpel/model/GatewayBranch.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2017 ZTE Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.winery.bpmn2bpel.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GatewayBranch {
```

Comment:
This class represents a branch following a gateway node.
It contains two properties, id is the first following node's id;  nodeList is the nodes of the branch order by flow sequence.

Sorry, I didnot find the proper term in BPMN2.0. I think a gateway node could be translated to a tree, so I called each of the following branch to be a gateway branch. Is that ok or could u give me some suggestions? thanks.